frontend/src/components/LogIn/login.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './login.module.css';
+import gmail_icon from '../Logos/gmail_icon.png';
+import fb_icon from '../Logos/fb_icon.png';
+import twtx_icon from '../Logos/twtx_icon.webp';
+import apple_icon from '../Logos/apple_icon.png';
+
+function EmailErrorMessage({username, isValid}) {
+    return <p className={styles.error_message}>Please enter a valid email.</p>
+}
+
+// NOTE: I'm gonna add this later after you can compare input to passwords set by 
+//    the users to see if they're correct
+//
+// function Password({password, isValid}) {
+//     return <p className={styles.error_message}>Incorrect password </p>
+// }
+
+export function LogIn() {
+    const [email, setEmail]=useState('')
+    var checkEmail= new RegExp("^.+@.+\\..+")
+    // checks email for @ and subsequent . 
+    const isEmailValid=email.length===0 || checkEmail.test(email)
+
+    return (
+        <div className={styles.login_group}>
+            <h1>LOGIN</h1>
+            <form>
+                <label>Email</label>
+                    <input type="text" placeholder="Email"onChange={(event)=>{
+                        const value = event.target.value;
+                        setEmail(value)
+                    }}/>
+                    {!isEmailValid && <EmailErrorMessage></EmailErrorMessage>}
+
+                <label>Password</label>
+                    <input type="text" placeholder="Password"/>
```

Comment:
You can change the attribute type to "password".  The characters typed into a password input are masked (typically with dots or asterisks) to prevent others from seeing the password.